src/colmap/geometry/gps.cc

```diff
@@ -32,6 +32,64 @@
 #include "colmap/math/math.h"
 
 namespace colmap {
+namespace utm {
+struct Params {
+  // The order of the series expansion, determining the precision.
+  static constexpr int order = 4;
+
+  // Powers of mathematical sign $n$, n[i] means $n^i$
+  std::array<double, order + 1> n;
+  // Alpha coefficients for the series expansion
+  std::array<double, order> alpha;
+  // Beta coefficients for the series expansion
+  std::array<double, order> beta;
+  // Delta coefficients for the series expansion
+  std::array<double, order> delta;
+
+  // Multiplicative factor
+  double A;
+  // UTM scale factor at the central meridian, typically 0.9996
+  const double k0 = 0.9996;
+  // Easting of the origin, typically 500 km
+  const double E0 = 500;
+  // Northing of the origin
+  double N0(double lat_or_hemi) const { return lat_or_hemi > 0 ? 0 : 1e4; }
+
+  Params() = delete;
```

Comment:
```suggestion
```
This should not be needed. The default constructor will be deleted by default already.